pkg/blockstorage/vmware/vmware.go

```diff
@@ -570,6 +629,36 @@ func (p *FcdProvider) getSnapshotTags(ctx context.Context, fullSnapshotID string
 
 // SnapshotsList is part of blockstorage.Provider
 func (p *FcdProvider) SnapshotsList(ctx context.Context, tags map[string]string) ([]*blockstorage.Snapshot, error) {
+	filterStr := generateDescription(tags)
+	if filterStr == "" {
+		return p.snapshotsListByTag(ctx, tags)
+	}
+
+	volumeIDs := strings.Split(tags[VolumeIdTag], ",")
+	if len(volumeIDs) == 0 {
+		log.Debug().Print("vSphere can't list by description without list of volumes. Cannot list snapshots")
+		return nil, nil
```

Comment:
This message would not be visible in K10 which discards debug.  I suggest defining a specific error and returning it.

Reply:
We have a similar approach in the previous ListSnapshot function, are you sure we need to introduce breaking changes here?